Bottom: db4cde58d70d76bf151e65924ac5bd0a2ed27671
Top:    2f07e40e9d56cefd92037bbecc846c6fa6a8cd1c
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-23 01:53:43 +1300

Refresh of streamresult-router

---

diff --git a/NEWS b/NEWS
index 796e948..61bad55 100644
--- a/NEWS
+++ b/NEWS
@@ -85,6 +85,10 @@ Improvements
 * ``PlaceHolder`` can now hold timestamps, and applies them before the test and
   then before the outcome. (Robert Collins)
 
+* ``StreamResultRouter`` added. This is useful for demultiplexing - e.g. for
+  partitioning analysis of events or sending feedback encapsulated in
+  StreamResult events back to their source. (Robert Collins)
+
 * The error message for setUp and tearDown upcall errors was broken on Python
   3.4. (Monty Taylor, Robert Collins, #1140688)
 
@@ -92,10 +96,6 @@ Improvements
   test that accidentally depended on the CPython repr for object().
   (Jonathan Lange)
 
-* ``StreamResultRouter`` added. This is useful for demultiplexing - e.g. for
-  partitioning analysis of events or sending feedback encapsulated in
-  StreamResult events back to their source. (Robert Collins)
-
 0.9.29
 ~~~~~~
